src/HasApiTokens.php

```diff
@@ -45,6 +45,9 @@ public function revokeAllAccessTokens(): void
     public function revokeToken(IAccessToken $token): void
     {
         $this->accessTokens->removeElement($token);
+        if ($this->accessToken === $token) {
+            $this->accessToken = null;
+        }
```

Comment:
In my opinion, an unnecessary one because we can always revoke the token and set it to null.